src/components/Chip/Chip.tsx

```diff
@@ -22,6 +22,29 @@ export type Props = PropsWithSpread<
      */
     appearance?: ValueOf<typeof ChipType>;
 
+    /**
+     * Whether the chip is read-only.
+     * If true, the chip will not be interactive.
+     */
+    isReadOnly?: boolean;
```

Comment:
Not sure if the chip without border is a good way to signal it being read only. We tend to use the non-bordered chip for inline in LXD-UI, but still being interactive:

<img width="347" height="68" alt="image" src="https://github.com/user-attachments/assets/ae19ae32-3038-455b-8587-28ec11961b4d" />

Maybe the read only variant can have a different style. Lower opacity or some other property changing to mark it as non-interactive. And maybe we should add the without border as a variant, like the is dense or is inline variant. Or maybe the is inline variant should automatically be without border. What do you think?

Reply:
Borderless (on top of no keyboard/mouse interactions) being used to indicate read-only comes from this [design spec](https://docs.google.com/document/d/1HBZ_i7nI0FQbR9FJ_SE0x3Ax_BUi4DnBXMEhZPp2BiI/edit?tab=t.5qfuxdsojs30#heading=h.co5ihleltffd).

The inline variant has already been in Vanilla for some time (just not explicitly documented in RC) - so we can't make it borderless by default without causing a visual change to people who are already using it - not sure if this would be considered breaking visually, wdyt @lyubomir-popov @dgtlntv ?

Am I to understand that you are using a chip styling override to remove the border for these cases, and aren't sure about if `is-readonly` is the right way to cover those use cases?

Reply:
> Am I to understand that you are using a chip styling override to remove the border for these cases, and aren't sure about if is-readonly is the right way to cover those use cases?

My concern is our UI already using a borderless chip which is interactive. Now, using borderless chips to indicate non-interactivity would be confusing.

Reply:
I am not sure borderless is a clear way to indicate no interactivity. Hence, my suggestion to change the read only design.